packages/neutrino/Neutrino.js

```diff
@@ -52,6 +32,18 @@ module.exports = class Neutrino {
       ...clone(opts)
     };
 
+    if ('node_modules' in options) {
+      throw new ConfigurationError(
+        'options.node_modules has been removed. Use `neutrino.config.resolve.modules` instead.'
+      );
+    }
+
+    if ('env' in options) {
```

Comment:
My bad for using the phrase `options.env` in the review comment previously - `env` was actually a top-level property and not under `options`. As such, this check doesn't catch existing usages any more. 

ie: Trying locally with `env` in `.neutrinorc.js` I don't get the `ConfigurationError`.